x-pack/test/saved_object_tagging/api_integration/security_and_spaces/apis/get_all.ts

```diff
@@ -50,10 +50,12 @@ export default function ({ getService }: FtrProviderContext) {
         },
       },
       unauthorized: {
-        httpCode: 200,
+        httpCode: 403,
         expectResponse: ({ body }) => {
           expect(body).to.eql({
-            tags: [],
+            error: 'Forbidden',
+            message: 'unauthorized',
+            statusCode: 403,
```

Comment:
`openPointInTimeForType` throws with a 403 when the user don't have the permissions to access any of the searched types, where `find` returns an empty list instead.  see https://github.com/elastic/kibana/pull/109062#discussion_r691051196 for more context